src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.model;
+
+public class RightTriangle extends Figure {
+    private int hight;
+    private int base;
+
+    public RightTriangle(String color, int hight, int base) {
+        super(color);
+        this.hight = hight;
+        this.base = base;
+    }
+
+    public int getHight() {
+        return hight;
+    }
+
+    public void setHight(int hight) {
+        this.hight = hight;
+    }
+
+    public int getBase() {
+        return base;
+    }
+
+    public void setBase(int base) {
+        this.base = base;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) hight * base / 2;
```

Comment:
hight * base / 2: integer division in floating-point context.
Please pay attention to [this](https://www.javatpoint.com/java-int-to-double) article.